### PR TITLE
Kotlin IO

### PR DESCRIPTION
**Pull request by JohanneA**
Added examples for FileTreeWalk and various File extension functions

### CODE REVIEW

**Review by jmfayard — APPROVED** [review]

**Comment by jmfayard** [comment]
Looks good to me, thanks  Johanne! @JohanneA

File: kotlin-jvm/src/main/resources/references/kotlinio/long-file.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+
+Bacon bresaola sirloin alcatra shoulder burgdoggen picanha frankfurter venison cow hamburger porchetta. Cow beef ribs frankfurter porchetta. Drumstick chislic cow, kevin tri-tip turkey pancetta ham t-bone hamburger pork belly filet mignon frankfurter. Ham picanha short loin filet mignon pancetta spare ribs frankfurter pork chop landjaeger sirloin flank jowl alcatra.
+
+Shank cow shoulder leberkas ham rump. Pork drumstick burgdoggen porchetta beef ribs. Meatloaf cow tenderloin tri-tip shankle kielbasa drumstick chuck ham hock turkey pancetta shank. Ribeye picanha salami, drumstick venison jerky tail flank shankle. Boudin tenderloin rump tongue bresaola ball tip tri-tip. Salami ham hock shankle hamburger pork chop boudin t-bone picanha cow pork loin capicola chicken ground round.
+
+Filet mignon corned beef short loin jerky pork belly. Shank shankle frankfurter porchetta, t-bone tail chuck salami pig hamburger turkey brisket. Boudin pig chuck, kevin ball tip ribeye biltong pork loin ham. Jerky cupim tenderloin shoulder turkey filet mignon. Turkey boudin buffalo tail biltong.
```
[the remark this answers is not in the document]
Comment (LouisCAD) [reply kotlin-jvm/src/main/resources/references/kotlinio/long-file.txt:9]: Can we have a vegetarian alternative? 😛

[the remark this answers is not in the document]
Comment (JohanneA) [reply kotlin-jvm/src/main/resources/references/kotlinio/long-file.txt:9]: That is actually possible https://veggieipsum.com/ 😁

[the remark this answers is not in the document]
Comment (LouisCAD) [reply kotlin-jvm/src/main/resources/references/kotlinio/long-file.txt:9]: Feel free to send an extra PR if you want to offer this option 😉

**Pull request merged**